informatiCup/informatiCup2024
# implement fake generator

implemented the fake image generator,

to test:
run the fake_image_generator class and check if a random image has been added to the ignored folder "fake_generator_images"
